README.md

```diff
@@ -3,12 +3,20 @@
 
 Terraform module for deploying Materialize on AWS Cloud Platform with all required infrastructure components.
 
-> **Warning** This is provided on a best-effort basis and Materialize cannot offer support for this module.
-
 The module has been tested with:
+
 - PostgreSQL 15
 - Materialize Helm Operator Terraform Module v0.1.8
 
+> [!WARNING]
+> This module is provided on a best-effort basis and Materialize cannot offer support for it.
+>
+> It is not guaranteed to be forward-compatible and may include breaking changes in future versions.
+>
+> The module is intended for demonstration and evaluation purposes only, not for production use.
```

Comment:
I'd start with this disclaimer, and then segue into the above. I think we're also saying it has two different purposes here (evaluation only) and later (use as starting point for prod infra), and as a reader I'm not quite sure what the actual intent is

maybe something like this?

"This module is intended to be used for demonstrations, simple evaluations, and as a template for building your own production deployment of Materialize on AWS.

This module should not be relied upon for production deployments directly: future releases of those module _will_ contain breaking changes. When used as a starting point for a production deployment, you _must_ fork this repo and pin to a specific version."

Reply:
cc @kay-kim same feedback for the disclaimer we have in the docs as well. I'm finding our current messaging reads a little confused. Seth had a nice clear take in https://materializeinc.slack.com/archives/C085QFLB072/p1740503361409879?thread_ts=1740502884.282719&cid=C085QFLB072 that I think is a good reference

Reply:
Sounds good! I prepared PRs to update the wording:

https://github.com/MaterializeInc/terraform-google-materialize/pull/31
https://github.com/MaterializeInc/terraform-azurerm-materialize/pull/23
https://github.com/MaterializeInc/terraform-aws-materialize/pull/51